Wrappers/Python/cil/utilities/dataexample.py

```diff
@@ -645,4 +353,184 @@ def scikit_random_noise(image, mode='gaussian', seed=None, clip=True, **kwargs):
+           The path to the data directory
+
+        Returns
+        -------
+        AcquisitionData
+        '''
+        loader = NEXUSDataReader()
+        loader.set_up(file_name=os.path.join(data_dir, cls.dfile))
+        return loader.read()
+class SYNCHROTRON_PARALLEL_BEAM_DATA(NEXUS_CIL_DATA):
+    '''A DLS dataset'''
+    dfile = '24737_fd_normalised.nxs'
+class SIMULATED_PARALLEL_BEAM_DATA(NEXUS_CIL_DATA):
+    '''A simulated parallel-beam dataset generated from SIMULATED_SPHERE_VOLUME'''
+    dfile = 'sim_parallel_beam.nxs'
+class SIMULATED_CONE_BEAM_DATA(NEXUS_CIL_DATA):
+    '''A cone-beam dataset generated from SIMULATED_SPHERE_VOLUME'''
+    dfile = 'sim_cone_beam.nxs'
+class SIMULATED_SPHERE_VOLUME(NEXUS_CIL_DATA):
+    '''A simulated volume of spheres'''
+    dfile = 'sim_volume.nxs'
+
+class WALNUT(_REMOTE_DATA):
+    '''A microcomputed tomography dataset of a walnut from https://zenodo.org/records/4822516'''
+    FOLDER = 'walnut'
+    URL = 'https://zenodo.org/record/4822516/files/walnut.zip'
+    FILE_SIZE = '6.4 GB'
+
+    @classmethod
+    def get(cls, data_dir):
+        '''
+        This function returns the raw projection data from the .txrm file
+
+        Parameters
+        ----------
+        data_dir: str
+           The path to the directory where the dataset is stored. Data can be downloaded with dataexample.WALNUT.download_data(data_dir)
+
+        Returns
+        -------
+        ImageData
+            The walnut dataset
+        '''
+        filepath = os.path.join(data_dir, cls.FOLDER, 'valnut','valnut_2014-03-21_643_28','tomo-A','valnut_tomo-A.txrm')
+        try:
+            loader = ZEISSDataReader(file_name=filepath)
+            return loader.read()
+        except FileNotFoundError as exc:
+            raise ValueError(f"Specify a different data_dir or download data with `{cls.__name__}.download_data({data_dir})`") from exc
+
+class USB(_REMOTE_DATA):
+    '''A microcomputed tomography dataset of a usb memory stick from https://zenodo.org/records/4822516'''
+    FOLDER = 'USB'
+    URL = 'https://zenodo.org/record/4822516/files/usb.zip'
+    FILE_SIZE = '3.2 GB'
+
+    @classmethod
+    def get(cls, data_dir):
+        '''
+        This function returns the raw projection data from the .txrm file
+
+        Parameters
+        ----------
+        data_dir: str
+           The path to the directory where the dataset is stored. Data can be downloaded with dataexample.USB.download_data(data_dir)
+
+        Returns
+        -------
+        ImageData
+            The usb dataset
+        '''
+        filepath = os.path.join(data_dir, cls.FOLDER, 'gruppe 4','gruppe 4_2014-03-20_1404_12','tomo-A','gruppe 4_tomo-A.txrm')
+        try:
+            loader = ZEISSDataReader(file_name=filepath)
+            return loader.read()
+        except FileNotFoundError as exc:
+            raise ValueError(f"Specify a different data_dir or download data with `{cls.__name__}.download_data({data_dir})`") from exc
```

Comment:
technically as these both use `ZEISSDataReader` they could inherit from a common intermediate class to reduce duplication.

However since it's only 2 classes it won't really reduce much duplication so I left it as-is.